sunscreen_compiler_common/src/context.rs

```diff
@@ -31,7 +37,11 @@ where
      * Creates a new [`NodeInfo`].
      */
     pub fn new(operation: O) -> Self {
```

Comment:
Oh, this will be a problem. You'll need to change this signature to `operation: O, group_id: usize` when the debugger feature is enabled. That will give you a bunch more errors to go fix.